spec/features/user_interacts_with_queue_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+feature "User interacts with the queue" do
+  scenario "user adds and reorders videos in the queue" do 
+    comedies = Fabricate(:category)
+    monk = Fabricate(:video, title: "Monk", category: comedies)
+    south_park = Fabricate(:video, title: "South Park", category: comedies)
+    futurama = Fabricate(:video, title: "Futurama", category: comedies)
+
+    sign_in
+
+    add_video_to_queue(monk)
+    expect_video_to_be_in_queue(monk)
+
+    visit video_path(monk)
+    expect_link_not_to_be_seen "+ My Queue"
+
+    add_video_to_queue(south_park)
+    add_video_to_queue(futurama)
+
+    set_video_position(monk, 3)
+    set_video_position(south_park, 1)
+    set_video_position(futurama, 2)
+    update_queue
+    
+    expect_video_position(south_park, 1)
+    expect_video_position(futurama, 2)
+    expect_video_position(monk, 3)
+  end
+
+  def expect_video_to_be_in_queue(video)
+    page.should have_content(video.title)
```

Comment:
You are using `should` syntax here while the name suggests `expect` RSpec syntax. I would recommend to stick with 1 style - either use one or another - don't mix them, it looks inconsistent. And here I would prefer using the latest `expect` syntax as thats the way RSpec team is going nowadays.